commit 4f50763b981f9056c5f1763dfb26cfa4a26a181d
Author: Jia Tan <jiat0218@gmail.com>
Date:   Thu Mar 16 21:44:02 2023 +0800

    CI: Add doxygen as a dependency.
    
    Autogen now requires --no-doxygen or having doxygen installed to run
    without errors.

diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index 19391ec4..8ce082d7 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -30,7 +30,7 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
       - name: Install Dependencies
-        run: sudo apt-get install -y autoconf automake build-essential po4a autopoint gcc-multilib
+        run: sudo apt-get install -y autoconf automake build-essential po4a autopoint gcc-multilib doxygen
 
       # -b specifies the build system to use.
       # -p specifies the phase (build or test) to help narrow down an error
@@ -117,7 +117,7 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
       - name: Install Dependencies
-        run: brew install autoconf automake libtool po4a
+        run: brew install autoconf automake libtool po4a doxygen
 
       - name: Build with full features
         run: ./build-aux/ci_build.sh -b autotools -p build
@@ -182,4 +182,3 @@ jobs:
         with:
           name: MacOS CMake Test Logs
           path: build-aux/artifacts
-    
\ No newline at end of file